pilot/pkg/model/push_context_test.go

```diff
@@ -22,40 +22,40 @@ import (
 func TestMergeUpdateRequest(t *testing.T) {
```

Comment:
Can you add a test case for the change you made

Reply:
Sure